Library/Homebrew/test/spec_helper.rb

```diff
@@ -38,6 +37,8 @@
 
 require_relative "../global"
 
+require "debug" if ENV["HOMEBREW_DEBUG"]
```

Comment:
i wonder if we should just do this for any command run with `--debug`?

Reply:
We ask end users to run `--debug` so I don't think so. Or if we do then it should catch a `LoadError` and fail silently.